src/apis/ProxyController.php

```diff
@@ -24,18 +27,33 @@
  * 4. Generate Build.
  * 5. Send build identifier to the client.
  *
+ * @property Module $module
+ *
  * @author Basil Suter <basil@nadar.io>
  * @since 1.0.0
  */
 class ProxyController extends Controller
 {
+    /**
+     * @var Connection
+     * @since 2.0.0
+     */
+    protected $db;
+
     /**
      * @var array A list of tables which will be ignored and can not be synced with the proxy command.
      */
     protected $ignoreTables = [
         'migration', 'admin_proxy_build', 'admin_proxy_machine',
     ];
     
+    public function init()
+    {
+        $this->db = Instance::ensure($this->db, Connection::className());
```

Comment:
Please use `class` over `className()`, as className() will be removed in next Yii version.